tests/selectors/settings.selectors.ts

```diff
@@ -5,3 +5,26 @@ export enum SettingsMenuSelectors {
   SwitchAccountItemIndex = 'switch-account-item-[index]',
   OpenWalletInNewTab = 'open-wallet-in-new-tab',
 }
+
+export enum SettingsSelectors {
```

Comment:
What is the difference here with `SettingsMenuSelectors`, can they be combined?

Reply:
hm, yeah. will combine